samples/excel/30-range/create-and-use-range-intersection.yaml

```diff
@@ -0,0 +1,154 @@
+id: excel-30-range-create-and-use-range-intersection
+name: Create and Use an Intersection of Ranges
+description: Create a an intersection of two ranges and make a chart of it.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#create-sales-contest-charts").click(() => tryCatch(createContestCharts));
+
+        async function createContestCharts() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");    
+                const salesTable = sheet.tables.getItem("SalesTable");
+                const dataRange = salesTable.getDataBodyRange();
+
+                // We want the most recent quarter that has data, so 
+                // exclude quarters without data and get the last of 
+                // the remaining columns.
+                const usedDataRange = dataRange.getUsedRange(true /* valuesOnly */);
+                const currentQuarterRange = usedDataRange.getLastColumn();
+
+                // Asian and European teams have separate contests.
+                const asianSalesRange = sheet.getRange("A2:E4");
+                const europeanSalesRange = sheet.getRange("A5:E7");
+
+                // The data for each chart is the intersection of 
+                // the current quarter column and the rows for the 
+                // continent.
+                const asianContestRange = asianSalesRange.getIntersectionOrNullObject(currentQuarterRange);
+                const europeanContestRange = europeanSalesRange.getIntersectionOrNullObject(currentQuarterRange);
+
+                // Must sync before you can test the output of *OrNullObject 
+                // method/property.
+                await context.sync();
+
+                if (asianContestRange.isNullObject) {
+                    // See the declaration of this method for how to 
+                    // test this code path.
+                    reportMissingData("Asian");
+                } else {
+                    createContinentChart(sheet, "Asian", asianContestRange, "A9", "F24");
+                }
+
+                if (europeanContestRange.isNullObject) {
+                    // See the declaration of this method for how to 
+                    // test this code path.
+                    reportMissingData("European");
+                } else {
+                    createContinentChart(sheet, "European", europeanContestRange, "A25", "F40");
+                }
+
+                await context.sync();
+            });
+        }
+
+        function createContinentChart(sheet: Excel.Worksheet, continent: string, contestRange: Excel.Range, startPosition: string, endPosition: string) {
```

Comment:
Script Lab itself breaks the line at a space (and even indents the continuation of the line). Precisely because it is so long, I think it helps readability to have that blank line after it.